napari/layers/base/base.py

```diff
@@ -1420,9 +1422,10 @@ def _get_ray_intersections(
 
     @property
     def _displayed_axes(self):
-        displayed_axes = [
-            self._dims_displayed[i] for i in self._dims_displayed_order
-        ]
+        # assignment upfront to avoid repeated computation of properties
+        _dims_displayed = self._dims_displayed
+        _dims_displayed_order = self._dims_displayed_order
+        displayed_axes = [_dims_displayed[i] for i in _dims_displayed_order]
```

Comment:
This was a separate change made to avoid repeated calls to the (computed) `self._dims_displayed` property in the list comprehension.